src/assets/locales/de/translations.json

```
@@ -106,17 +106,17 @@
   "about-us.title": "Über uns",
   "about-us.office.title": "Office",
   "about-us.office.kicker": "Satellytes",
-  "about-us.office.heading": "Unser neues Office direkt am Sendlinger Tor",
+  "about-us.office.heading": "Im Herzen von München - direkt am Sendlinger Tor",
   "about-us.office.text": "Fühlen Sie sich bei uns wie zu Hause. Ein paar kleine Eindrücke unseres neuen Büros in der Sendlinger Straße im Herzen Münchens.",
   "about-us.team.title": "Team",
   "about-us.team.heading": "Das sind wir",
-  "about-us.team.text": "Unser erfahrenes Team aus Programmierern und Designern",
+  "about-us.team.text": "Unser erfahrenes Team aus Software Ingenieuren und Designern",
   "about-us.leadbox.title": "Haben Sie Interesse bekommen?",
   "about-us.leadbox.link": "Karriere",
   "services.hero": "Leistungen",
-  "services.title": "Unser Angebot an Sie",
+  "services.title": "Wir verbinden Development & Design",
   "services.kicker": "Plattformen, Produkt & Beratung",
-  "services.text": "Wir haben großen Spaß an Technologie und freuen uns auf neue Herausforderungen. Dabei fokussieren wir uns auf langfristige Engagements von Mittelstand bis Konzerne. Neuen Aufgaben begegnen wir immer mit angemessenem Respekt. Wir streben stets hochwertige und zeitgemäße Lösungen an – die Wahl der Technologie ist für uns dabei sekundär. Gern unterstützen wir auch Sie in den folgenden Disziplinen.",
+  "services.text": "Unsere Software Ingenieure und Designer vereint eines: die Liebe zum Detail und zur Technologie. Beide bringen wir für die Ziele Ihres Unternehmens und die Experience Ihrer Kunden ein. So finden wir innovative und sichere Lösungen. Mit Respekt, Neugierde, Kreativität. Und jeder Menge Herzblut.",
```

This time without the strange unicode character but I'd like to suggest to combine the last two sentences here as well:
```suggestion
  "services.text": "Unsere Software Ingenieure und Designer vereint eines: die Liebe zum Detail und zur Technologie. Beide bringen wir für die Ziele Ihres Unternehmens und die Experience Ihrer Kunden ein. So finden wir innovative und sichere Lösungen. Mit Respekt, Neugierde, Kreativität und jeder Menge Herzblut.",
```